nginx/server/sites.sls

```diff
@@ -50,29 +56,23 @@
   - require:
     - pkg: nginx_packages
 
-nginx_init_{{ site.host.name }}_tls:
-  cmd.wait:
-  - name: "cat /etc/ssl/certs/{{ site.host.name }}.crt /etc/ssl/certs/{{ site.host.name }}-ca-chain.crt > /etc/ssl/certs/{{ site.host.name }}-with-chain.crt"
-  - watch:
-    - file: /etc/ssl/certs/{{ site.host.name }}.crt
-    - file: /etc/ssl/certs/{{ site.host.name }}-ca-chain.crt
-  - watch_in:
-    - service: nginx_service
-
-{%- endif %}
+{% endif %}
 
-{%- endif %}
+{% endif %}
 
-{%- elif site.ssl.engine == 'salt' %}
```

Comment:
this may broke something, but still *_file allows to fully configure engine to create expected chain file.

originaly part of: 
```
{%- elif site.ssl.engine == 'salt' %}
```